src/mongo/db/storage/named_pipe.h

```diff
@@ -37,6 +37,7 @@
 #include <string>
 
 #include "mongo/db/storage/default_path.h"
+#include "mongo/db/storage/error_count.h"
```

Comment:
[nit] This does not seem to be necessary anymore.

Reply:
Done